basic:
  use_compressed_image: true
  camera_topic: /camera/color/image_raw/compressed
  detect_every_k_frames: 3

cam:
  cam_net_type: "YOLOV4_TINY"
  cam_file_model_cfg: "/asset/yolov4-tiny.cfg"
  cam_file_model_weights: "/asset/yolov4-tiny.weights"
  cam_inference_precison: "FP32"
  cam_n_max_batch: 1
  cam_prob_threshold: 0.5
  cam_min_width: 50
  cam_max_width: 1920
  cam_min_height: 50
  cam_max_height: 1080
